### PR TITLE
fix(FEC-9269) - networkConnectionType is missing in PLAY event

### PR DESCRIPTION
**Pull request by RoyBregman**
added the networkConnectionType to play event + unit tests

### CODE REVIEW

File: src/kava.js [hunk 1]
```diff
@@ -413,6 +413,12 @@ class Kava extends BasePlugin {
     }
   }
 
+  _networkConnectionType(): string {
```
Comment (OrenMe) [inline src/kava.js]:
```suggestion
  _getNetworkConnectionType(): string {
```

Comment (RoyBregman) [reply src/kava.js]: I guess it would be possible but I would do it in the tidbits ticket and not now

Comment (OrenMe) [reply src/kava.js]: ok, please fix naming.
Add the change to the ticket if it isn’t there already.

**Comment by RoyBregman** [comment]
> Flow is failing

**Pull request merged**